sample-client-app/sample-app-with-shared-lib/build.gradle

```diff
@@ -4,7 +4,7 @@
  */
 apply plugin: 'eclipse'
 apply plugin: 'java'
-project.group = 'com.vmware.sample'
+project.group = 'com.vmware.vipclient'
```

Comment:
Is it better set the group as 'com.vmware.singleton' ?

Reply:
Actually, project.gorup does not matter for this as we are not publishing this artifact anywhere. To minimize the changes, I am reverting this line to the original.